.github/workflows/pr.yaml

```diff
@@ -0,0 +1,79 @@
+name: run
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  stage_pr:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: 'read'
+      pull-requests: 'write'
+      id-token: 'write'
+
+    steps:
+    - uses: actions/checkout@v2
+      name: 'Check out Github repository'
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+    - run: cd web ; npm install ; npm run build
+  
+
+    # Install gcloud, do not specify authentication.
+    - uses: 'google-github-actions/setup-gcloud@v0'
+      name: 'Set up Google Cloud SDK'
+      with:
+        project_id: ${{ secrets.GCP_PROJECT }}
+
+    # Configure Login via a service account key secret.
+    - id: 'auth'
+      name: 'Authenticate to Google Cloud using Service Account Key secret'
+      uses: 'google-github-actions/auth@v0'
+      with:
+        credentials_json: '${{ secrets.GCP_CREDENTIALS }}'
+
+    - uses: imjasonh/setup-ko@v0.4
+      name: 'Set up ko'
+      id: ko
+    - env:
+        KO_DOCKER_REPO: 'gcr.io/gcping-devrel'
+      name: 'Publish to Google Container Registry'
+      run: |-
+        gcloud auth configure-docker gcr.io --quiet
+        echo IMAGE=$(ko publish ./cmd/ping/) >> $GITHUB_ENV 
```

Comment:
Maybe I missed it, but are we setting `KO_DOCKER_REPO`?

Reply:
Yes:
https://github.com/GoogleCloudPlatform/gcping/blob/d3f5399cb87760f5bd39ecad9701fd6e6b4a8390/.github/workflows/pr.yaml#L41-L42

Reply:
OK. Thanks!